tool.ts

```diff
@@ -84,33 +103,45 @@ export function evaluateVersions(versions: string[], versionSpec: string): strin
 /**
  * finds the path to a tool in the local installed tool cache
  * 
- * @param toolName  name of the tool
- * @param version   version to get the path of
- * @param arch      optional arch.  defaults to arch of computer
+ * @param toolName      name of the tool
+ * @param versionSpec   version of the tool
+ * @param arch          optional arch.  defaults to arch of computer
  */
-export function findLocalTool(toolName: string, version: string, arch?: string) {
+export function findLocalTool(toolName: string, versionSpec: string, arch?: string): string {
```

Comment:
I changed findLocalTool to expose versionSpec. This eliminates the caller needing to findLocalToolVersions, evaluateVersions, and finally findLocalTool with the explicit version.

This seemed appropriate to me, although diverges from the original pattern set forth. Let me know if I'm overlooking something here and I can revert to the original pattern.